<?xml version="1.0"?>
<testsuites>
 <testsuite name="BurgerShot_script" package="BurgerShot.suite" timestamp="2019-06-23 21:14:40.125 -0500"
 tests="1"  failures="3" errors="0" time="4.714000">
  <properties>
   <property name="RunDate" value="2019-06-23 21:14:40.125 -0500"/>
   <property name="Status" value="Failure"/>
  </properties>
  <testcase name="BurgerShot.script" assertions="2" status="failure"  time="4.703">
    <failure type="LogError" message="Exception encountered at: line 34 of burgershot" />
    <failure type="LogError" message="Exception reason: Bad value for Object expression.  Looking for an object or script identifier, got 'Setup/Task'." />
    <failure type="FAILURE" message="Script Error Number of Errors logged: 2 -- Execution Time 0:00:04 BurgerShot.script" />
  </testcase>
 </testsuite>
</testsuites>
